test/IntegrationTests.js

```diff
@@ -625,15 +625,15 @@ tap.test('getTransactions should send options and return the transactions', func
 
   server.on('request', function (req, res) {
     t.equal(req.method, 'GET')
-    t.equal(req.url, '/api/1/transactions?asset=XBT&offset=5&limit=5')
+    t.equal(req.url, '/api/1/accounts/1224342323/transactions?min_row=5&max_row=5')
     res.end(JSON.stringify(expectedTransactions))
   })
 
   var options = {
-    offset: 5,
-    limit: 5
+    min_row: 5,
+    max_row: 5
```

Comment:
Please use different values for the min and max so that this test will fail if the logic gets accidentaly inverted at some point.